themes/vscode-light-plus.json

```diff
@@ -52,7 +52,7 @@
         "panel.indent_guide_hover": "#e4e6f1",
         // Scrollbar
         "scrollbar.thumb.background": "#dddddd",
-        "scrollbar.thumb.hover_background": "#f00",
+        "scrollbar.thumb.hover_background": "#d1d1d1",
```

Comment:
Was this just a personal choice or did you compare this to the VSCode version at all? It's sometimes a balance of both but I'm just trying to gauge where this is coming from.